packages/i18n/src/resources/ru.json

```diff
@@ -9,15 +9,19 @@
   "Check Our Docs": "Проверьте нашу документацию",
   "is a Cloud Workspace": "это облачное рабочее пространство.",
   "Owner": "Владелец",
-  "Export AFFiNE backup file": "Экспорт файла резервной копии AFFiNE",
+  "Export AFFiNE backup file": "Экспорт файла резервной копии AFFiNE (скоро)",
   "is a Local Workspace": "это локальное рабочее пространство",
   "Wait for Sync": "Дождитесь синхронизации",
   "Joined Workspace": "Присоединенное рабочее пространство",
   "Retain local cached data": "Сохранять локальные кэшированные данные",
   "will delete member": "удалит участника",
   "Download all data": "Скачать все данные",
-  "It takes up little space on your device": "Занимает мало места на вашем устройстве.",
-  "It takes up more space on your device": "Занимает много места на вашем устройстве.",
+  "It takes up little space on your device": {
+    "": "Занимает мало места на вашем устройстве."
```

Comment:
☝️

Reply:
I am currently working on resolving this issue.